draft-ietf-quic-spin-exp.md

```diff
@@ -235,10 +235,16 @@ Implementations SHOULD allow administrators of clients and servers to disable
 the spin bit either globally or on a per-connection basis.
 Even when the spin bit is not disabled by the administrator implementations
 SHOULD disable the spin bit on a randomly chosen
-fraction of connections.  The selection process should be designed such that
-on average the spin bit is disabled for at least 1/8th of the connections.
+fraction of connections.
 
-When the spin bit is disabled, endpoints SHOULD set the spin bit value to zero,
+The selection process SHOULD be designed such that
+on average the spin bit is disabled for at least 1/8th of the connections, or
+1/8th of the paths when doing migrations. The random choice SHOULD be dependent
+on the address of the peer, so that the spin bit is consistently enables or
+disabled for repeated connections to the same address.
+
```

Comment:
That won't work well in server p2p where many connections go through the same 5-tuple because either all or no traffic will get spins. It will also will not work well when a cluster of servers target the same IP using the same PRNG /hash algorithm. You need to depend on the source address as well.

I think 1/8th of CIDs would be better.

Reply:
It does seem like after reading the discussion in issue #1828 that spinning per CID would be better.

Reply:
But if you include the source address as well, doesn't that mean that most clients would ordinarily spin and a server that gets spin from clients at a near-zero rate remains suspicious?

Reply:
@huitema this makes sense to me.